### PR TITLE
Moneropedia entries (E–O)

### PR DESCRIPTION
**Pull request by MindWallet**

### CODE REVIEW

**Comment by erciccione** [comment]
See #1309

File: _i18n/nb-no/resources/moneropedia/encryption.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+### Kovri
+
+Kovri implementerer ulike typer kryptering i *minst* fire essensielle kapasiteter:
+
+- Reseed for bootstrapping
+- Garlic-routing: tre lag med kryptering («garlic-encryption») brukes for å verifisere den sikre leveringen av meldinger til mottakeren/motparten/destinasjonen
+- Tunnelkryptering: garlic-meldinger passeres gjennom en tunnel og krypteres av tunnelportalen til tunnelsluttpunktet
+- Transportlagkryptering hindrer muligheten til å avkode meldinger på [medialag](https://en.wikipedia.org/wiki/OSI_model)
+
+For detaljer om krypteringstyper og kryptografiske signaturer som er brukt i Kovri og I2P, kan du besøke Java-I2Ps artikkel om [kryptografi](https://geti2p.net/spec/cryptography)
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/encryption.md:34]: besøke -> se*

File: _i18n/nb-no/resources/moneropedia/fungibility.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+terms: ["fungibility", "fungible", "fungibilitet", "fungibelt"]
+summary: "en egenskap som en krytovaluta har der to enheter kan byttes med hverandre"
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/fungibility.md:3]: typo

File: _i18n/nb-no/resources/moneropedia/fungibility.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+Fungibilitet er en fordel som Monero har over Bitcoin og nesten alle andre kryptovalutaer grunnet det iboende personvernet i Monero-blokkjeden og den vedvarende, sporbare naturen ved Bitcoin-blokkjeden. Med Bitcoin kan enhver BTC spores tilbake til alle helt til den første transaksjonen. Hvis en mynt tidligere har blitt brukt til et ulovlig formål, vil denne historikken derfor inneholdes i blokkjeden for alltid. Denne mangelen på fungibilitet betyr at visse foretak vil være forpliktet til å unngå å godta BTC som tidligere har blitt brukt for ulovlige formål så de ikke bryter med tjenestevilkårene sine. Det er for tiden noen store Bitcoin-selskaper som blokkerer, suspenderer eller stenger kontoer som har mottatt Bitcoin som har blitt brukt i gambling på nett eller andre formål som er ansett som tvilsomme av disse selskapene.
+
+Monero har blitt bygget for å spesifikt ta stilling til sporbarhetsproblemet og ikke-fungibiliteten som er iboende i andre kryptovalutaer. Ved å ha fullstendig private transaksjoner, er Monero fullstendig fungibelt – (og således kan ingen XMR svartelistes), mens det samtidig gir alle fordelene av en sikker, desentralisert, permanent blokkjede.
+
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/fungibility.md:16]: Remove extra space

File: _i18n/nb-no/resources/moneropedia/kovri.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+terms: ["Kovri"]
+summary: "C++-ruterimplementering av I2P-nettverket"
+---
+
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Det grunnleggende
+
+[Kovri](https://gitlab.com/kovri-project/kovri/) er en C++-implementering av I2P-nettverket som ble foreslått integrert i Monero. Per i dag er fremtiden til denne I2P-ruteren uklar, og en integrasjon med Monero er ikke planlagt.
+
+Les [anonimals FFS-forslag](https://forum.getmonero.org/9/work-in-progress/86967/anonimal-s-kovri-full-time-development-funding-thread) for flere detaljer og for resonnementetet bak prosjektet. Les også OSS-siden og brukerveiledningen i [Kovri-arkivet](https://gitlab.com/kovri-project/kovri/).
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/kovri.md:12]: typo in resonnementetet

File: _i18n/nb-no/resources/moneropedia/locally-unique-host.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+### Inngående informasjon
+
+Du har muligheten til å dele din tolkning av hvordan verten blir oppløst (`localhost` oppløses for eksempel alltid til `127.0.0.1`), men resolusjonen fullbyrdes ikke kanonisk (noen andre kan for eksempel mappe `localhost` til en vilkårlig IP-adresse).
+
+Verter i et offentlig abonnement kan betraktes som kanonisk unike verter i I2P-nettverket, men du står til syvende og sist fritt til å omdefinere dem slik du ønsker.
+
+### Merknader
+
+- Monero bruker i hovedsak kanonisk unik vert-oppløsning, mens I2P kun bruker lokal unik vert-oppløsning.
+- I2P-er og @Kovrier som er tilordnet toppnivådomener er for tiden `.i2p`, og Kovri har kun til hensikt å behandle/bruke `.i2p`-[toppnivådomenet](https://en.wikipedia.org/wiki/Top_level_domain)
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/locally-unique-host.md:21]: Remove @

File: _i18n/nb-no/resources/moneropedia/mining.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+terms: ["mining", "miner", "miners", "utvinning", "utvinner", "utvinnere"]
+summary: "prosessen av å kryptografisk beregne et matematisk bevis for en blokk som inneholder en rekke transaksjoner, som deretter legges til blokkjeden"
+
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/mining.md:4]: remove extra space

File: _i18n/nb-no/resources/moneropedia/mining.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+---
+ 
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Det grunnleggende
+
+Prosessen av å kryptografisk beregne et matematisk bevis for en blokk som inneholder en rekke transaksjoner, som deretter legges til blokkjeden.
+
+Utvinning er den distribuerte prosessen av å bekrefte transaksjoner på den offentlige hovedboken som ikkeholder alle transaksjonene, altså blokkjeden. Monero-noder bruker blokkjeden til å skille legitime transaksjoner fra forsøk på å bruke mynter om igjen som allerede har blitt brukt et annet sted.
+
+Monero er utelukkende drevet av «Proof of Work» («bevis på arbeid»). Den bruker en utvinningsalgoritme som potensielt kan tilordnes milliarder av enheter (alle moderne x86-CPU-er og GPU-er). Monero bruker en variant av CryptoNight Proof of Work (PoW)-algoritmen, som er designet for å brukes i vanlige CPU-er og GPU-er.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/mining.md:14]: .. og mange* GPU-er

File: _i18n/nb-no/resources/moneropedia/openalias.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
+---
+terms: ["OpenAlias"]
+summary: "en standard som lar deg bruke en e-post eller domenesyntaks til å betale noen istedenfor en adresse, f.eks. donate@getmonero.org eller donate.getmonero.org"
+---
+
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Det grunnleggende
+
+Monero-kjerneteamet lanserte en standard, kalt OpenAlias, som tillater mer lettleste adresser, og Zookos trekant «til en firkant». OpenAlias kan brukes for enhver kryptovaluta, og er allerede implementert i Monero, Bitcoin (i den seneste Electrum-versjonen) og HyperStake.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/openalias.md:9]: ..og gjør Zookos*

File: _i18n/nb-no/resources/moneropedia/openalias.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+OpenAlias prøver å gi en måte å forenkle oppretting og bruk av aliser i et teknologiklima som stadig er i endring. Brukere prøver å krysse broen til private og kryptografisk sikre infrastrukturer og systemer, men mange av dem har såvidt begynte å huske e-postadressene til vennene og familien sin.
+
+Som del av den pågående utviklingen av Monero-kryptovalutaprosjektet, spurte vi oss selv: Hvordan kan vi simplifisere betalinger for brukere som ikke er vant med krytovaluta? Moneros stealth-adresser har en lengde på minst 95 tegn – å huske dem er ikke er alternativ, og å spørre noen om å sende en betaling til en <95-character-string> vil bare føre til forvirring.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/openalias.md:13]: Perhaps replace simplifisere with forenkle

File: _i18n/nb-no/resources/moneropedia/node.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+Reglene som nodene følger er bygget inn i Monero-programvaren. Når alle noder samtykker i reglene som skal følges, kalles det konsensus. Konsensus er nødvendig for en kryptovaluta fordi det er hvordan blokkjeden er bygget opp; Dersom noder ikke er enige om hvilke blokker som er gyldige – for eksempel de som ikke har oppdatert Monero-programvaren sin – vil de nodene ikke kunne delta i Monero-nettverket.
+
+Monero-kjerneteamet har som plan å gjennomføre en nettverksoppgradering omtrent hver 6. måned. Hvis du på det tidspunktet kjører en node, må den oppdateres til siste versjonen av Monero-programvaren, ellers vil den ikke lenger kunne delta i nettverket.
+
+---
+
+##### Andre ressurser
+<sub>1. *Fluffypony gir en fin forklaring på hvorfor obligatoriske nettverksoppgraderinger er bra for Monero.* ([Monero Missives for the Week of 2016-06-20](https://getmonero.org/2016/06/20/monero-missive-for-the-week-of-2016-06-20.html))</sub>
```
Comment (erciccione) [inline _i18n/nb-no/resources/moneropedia/node.md:24]:
```suggestion
<sub>1. *Fluffypony gir en fin forklaring på hvorfor obligatoriske nettverksoppgraderinger er bra for Monero.* ([Monero Missives for the Week of 2016-06-20]({{ site.baseurl_root }}/2016/06/20/monero-missive-for-the-week-of-2016-06-20.html))</sub>
```

This is not related to your translation, but i see it now, would be good to fix it. I'll take care of the other occurrences.

Comment (erciccione) [reply _i18n/nb-no/resources/moneropedia/node.md]: Fixed for the other languages in #1364

**Review by erciccione — APPROVED** [review]

**Pull request merged**